client/cmd/dexcctl/simnet-setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# Set up DCR and BTC wallets and register with the DEX.
+# dcrdex, dexc, and the wallet simnet harnesses should all be running before
+# calling this script.
+set -e
+echo initializing
+./dexcctl -p abc init
```

Comment:
Recently I've been running dexc with `--rpcaddr=localhost:6757 --webaddr=localhost:6758` for simnet, which goes back to https://github.com/decred/dcrdex/issues/557.  This makes me realize the network dependent ports would need to be coordinated with decctl.  That is, do you think it would make sense to have a --testnet/--simnet flag for dexcctl that switches these ports automatically?